packages/notification-center/src/components/notification-center-web-component/notification-center-component.types.ts

```diff
@@ -25,6 +25,7 @@ interface PopoverAdditionalProps {
   unseenCountChanged?: IPopoverNotificationCenterProps['onUnseenCountChanged'];
   actionClicked?: IPopoverNotificationCenterProps['onActionClick'];
   tabClicked?: IPopoverNotificationCenterProps['onTabClick'];
+  preferenceFilter?: IPopoverNotificationCenterProps['preferenceFilter'];
```

Comment:
I feel this should be added to the pick in `SelectedPopoverProps`  (line 31) instead of here. As these are for the unique renaming of our functions

Reply:
looks like i missed this one thanks!